README.md

```diff
@@ -28,6 +29,9 @@ This Sprint Challenge is split into three parts:
 
 #### Task 1. Implement a Ring Buffer Data Structure
 
+## This was a little challenging to grasp at first and very few resources online. Thinking of it as a VCR tape makes sense in that the VCR only has the capacity of length of the tape. Once tape capacity has been reached, rewind to beginning if you want new material added. So on and so forth which the recording would erase over the previous material from beginning until end.
```

Comment:
Hahahah, I thought of a tape desk/cassette tape player; this is the same idea! I guess we're all getting older HAHAHHAHAHHA!